files/ru/web/api/push_api/index.html

```diff
@@ -10,38 +10,38 @@
 ---
 <p>{{DefaultAPISidebar("Push API")}}{{SeeCompatTable}}</p>
 
-<p><strong>Push API</strong> даёт веб приложениям возможность получать сообщения отправленные с сервера, независимо от того, запущено ли веб приложение в фоне или даже загружено ли оно вообще пользователем. Это позволяет разработчикам посылать асинхронные сообщения и обновления пользователям которые подписались на них, как результат улучшается информированность пользователей о новом контенте</p>
+<p><strong>Push API</strong> даёт веб приложениям возможность получать сообщения, отправленные с сервера, независимо от того, запущено ли веб приложение в фоне или даже загружено ли оно вообще пользователем. Это позволяет разработчикам посылать асинхронные сообщения и обновления пользователям которые подписались на них и как результат — улучшается информированность пользователей о новом контенте.</p>
 
 <div class="note">
-<p><strong>Примечание</strong>: Эта документация охватывает спецификацию W3C Push API ; если вы ищете документацию по запатентованной технологии Firefox OS , <a href="/en-US/docs/Web/API/Simple_Push_API">смотрите здесь</a>.</p>
+<p><strong>Примечание</strong>: Эта документация охватывает спецификацию W3C Push API; если вы ищете документацию по запатентованной технологии Firefox OS , <a href="/en-US/docs/Web/API/Simple_Push_API">смотрите здесь (архивная страница)</a>.</p>
 </div>
 
 <h2 id="Общие_сведения_об_использовании_push_уведомлений">Общие сведения об использовании push уведомлений</h2>
 
 <div class="warning">
-<p><span id="result_box" lang="ru"><span>При внедрении подписки PushManager, жизненно важно защищать запросы от CSRF/XSRF атак в вашем приложении.</span> </span><span lang="ru"><span> </span></span></p>
+<p>При внедрении подписки PushManager, жизненно важно защищать запросы от CSRF/XSRF атак в вашем приложении.</p>
 
-<p><span lang="ru"><span>Дополнительную информацию см. В следующих статьях:</span></span></p>
+<p>Дополнительную информацию см. В следующих статьях:</p>
 
 <ul>
  <li><a href="https://habr.com/post/318748/">Методы защиты от CSRF-атаки</a></li>
- <li><a href="https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)_Prevention_Cheat_Sheet">Site Request Forgery (CSRF) Prevention Cheat Sheet</a></li>
- <li><a href="/ru/docs/https://blog.codinghorror.com/preventing-csrf-and-xsrf-attacks/">Preventing CSRF and XSRF Attacks</a></li>
+ <li><a href="https://www.owasp.org/index.php/Cross-Site_Request_Forgery_(CSRF)_Prevention_Cheat_Sheet">Шпаргалка по предотвращению подделки запросов к сайту (CSRF)</a></li>
```

Comment:
Предлагаю обновить ссылку в соответствии с англоязыной версией:
```suggestion
 <li><a href="https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html">Шпаргалка по предотвращению подделки запросов к сайту (CSRF)</a></li>
```